designer/server/src/test/scala/pl/touk/nussknacker/ui/api/helpers/NuResourcesTest.scala

```diff
@@ -118,14 +118,16 @@ trait NuResourcesTest
 
   protected val featureTogglesConfig: FeatureTogglesConfig = FeatureTogglesConfig.create(testConfig)
 
-  protected val typeToConfig: ProcessingTypeDataProvider[ProcessingTypeData, _] =
+  protected val typeToConfig: ProcessingTypeDataProvider[ProcessingTypeData, _] = {
+    implicit val stefan: DeploymentService = deploymentService()
```

Comment:
Please remove the stefan :D

Reply:
Fixed. Stefan is no longer working with us.